frontend/src/app/events/pages/event/event.page.ts

```diff
@@ -54,5 +56,19 @@ export class EventPage implements OnInit, OnDestroy{
     });
     return await loading.present();
   }
+  async presentAlert(errMessage) {
+    const alert = await this.alertController.create({
+        header: 'Wait..',
+        subHeader: 'The event couldn\'t be loaded',
+        message: 'We cannot get the event information: ' + errMessage,
+        buttons: ['Close']
+      });
+    await alert.present();
+  }
+  private handleError(error: HttpErrorResponse) {
+
+    this.presentAlert(`${JSON.stringify(error.error)}`);
```

Comment:
We need to have a specific **error field** that is agreed with the **backend team**. Is this settled with **backend** team?